### DISCUSSION TITLE
¿Donde se suben las practicas de Front End?

### DISCUSSION THREAD

**Discussion by floressdi** [Launch-X-Latam/MisionFrontEnd#453] [Team Posts]
Hola Explorers!! 
Tengo un repositorio con la practica 1 de Front End pero donde se suben aparte del repositorio que tenemos saludos!!

**Comment by JoshuaPerez22** [comment]
Hola! 
Por lo que he entendido con eso basta por ahora.

**Comment by mafer23** [comment]
![image](https://user-images.githubusercontent.com/39041288/196169970-b3664bbc-3039-42d8-8df9-ab5de7827600.png)

lo unico que se puede pensar

**Comment by jjmillar** [comment]
> lo unico que se puede pensar

Hola! Oie la verdad no tiene mucho sentido hacer un fork del repositorio de launch X. Entendiendo que lo que sugieres sirve para modificar archivos del repo de Launch-X y ese pull request es para eso efectivamente, modificar los archivos del repo, pero no es lo que queremos lograr. Para subir las practicas ya han dicho en casi todos los video que simplemente nos creemos un repo personal de practicas y listo, eso es todo.

**Comment by yomidev** [comment]
Que tal compañeros,

Creo que no debemos hacer lo del fork y el pull request ya que en la practica 2 de HTML, en las instrucciones el Mission Commander menciona que tenemos hasta el 4 de Noviembre para terminarlas y que ese mismo día va a habilitar un formulario para que coloquemos los links correspondientes de los repositorios en donde subimos nuestras practicas.

Por el momento solo debemos de hacerlas digámoslo así en nuestro perfil de Github

Saludos

**Comment by gumodi35** [comment]
Al finalizar el contenido del curso de **Frontend**, se habilita un link para que adjunten la liga de su repositorio y sus practicas, 📚  la idea es que  uds. vayan creando su **portfolio** con las practicas que realizaron en el curso, asi se hizo en la edición anterior y no creo que vaya haber cambios, así que basta con que en su 👨‍🏫  perfil de github adjunten las practicas, si hay modificaciones se hará saber por parte del **Mission Comander**. Saludos!! 👍

**Comment by floressdi** [comment]
¡Muchas Gracias por sus respuestas Explorers!

**Comment by Annamalag** [comment]
Siii súper ayudan a los que andamos colgados.. gracias totales a

**Comment by lopez-juandavid** [comment]
> Al finalizar el contenido del curso de Frontend, se habilita un link para que adjunten la liga de su repositorio y sus practicas, 📚 la idea es que uds. vayan creando su portfolio con las practicas que realizaron en el curso, asi se hizo en la edición anterior y no creo que vaya haber cambios, así que basta con que en su 👨‍🏫 perfil de github adjunten las practicas, si hay modificaciones se hará saber por parte del Mission Comander. Saludos!! 👍

Buenas explorer, entonces el veredicto final es que creemos un repositorio de FRONT END aparte, con todo los documentos entregables y esperemos que habiliten en link para subirlo por ahí?

**Comment by eduardo-lopez-91** [comment]
> Al finalizar el contenido del curso de Frontend, se habilita un link para que adjunten la liga de su repositorio y sus practicas, books la idea es que uds. vayan creando su portfolio con las practicas que realizaron en el curso, asi se hizo en la edición anterior y no creo que vaya haber cambios, así que basta con que en su man_teacher perfil de github adjunten las practicas, si hay modificaciones se hará saber por parte del Mission Comander. Saludos!! +1

Muchas Gracias, suena más lógico.
Apenas ando poniendome al corriente con la info y las prácticas.
Suerte a todos explorers
